### PR TITLE
fix(locale): fixed version log language switching

### PR DESCRIPTION
**Pull request by Qi-Cui**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

**Review by nannan00 — APPROVED** [review]
LGTM

File: src/bk-user/bkuser/apis/web/version_log/file_extractor.py [hunk 1]
```diff
@@ -31,7 +31,12 @@ def _read_file_content(file_path: str) -> str:
 
 def _get_change_log_file_name() -> str:
     """获取日志文件名称"""
-    if translation.get_language() == BkLanguageEnum.EN.value:
+
+    # 如果语言名中含有“-”，则取第一部分与 BkLanguageEnum 比对，例如 en-US 则取 en
+    lang = translation.get_language()
+    if "-" in lang:
+        lang = lang.split("-")[0]
+    if lang == BkLanguageEnum.EN.value:
```
Comment (narasux) [inline src/bk-user/bkuser/apis/web/version_log/file_extractor.py:39]: image

Comment (narasux) [reply src/bk-user/bkuser/apis/web/version_log/file_extractor.py:39]: btw BkLanguageEnum 继承 str，可以直接 `if lang == BkLanguageEnum.EN:`

File: src/bk-user/tests/apis/web/version_log/test_version_log.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    def test_valid_language_code_en(self, language):
+        translation.get_language = lambda: language
+        assert _get_change_log_file_name() == FILE_NAME_EN
+
+    @pytest.mark.parametrize(
+        ("language"),
+        [
+            ("zh"),
+            ("fr"),
+            ("zh-CN"),
```
Comment (narasux) [inline src/bk-user/tests/apis/web/version_log/test_version_log.py:34]: Nit: 现在枚举值是全小写

Comment (Qi-Cui) [reply src/bk-user/tests/apis/web/version_log/test_version_log.py]: done

File: src/bk-user/bkuser/apis/web/version_log/file_extractor.py (hunk 1 above)
Comment (Qi-Cui) [reply src/bk-user/bkuser/apis/web/version_log/file_extractor.py]: done

**Review by narasux — APPROVED** [review]

**Pull request merged**